docs/source/developer_guide/continuous_integration.rst

```diff
@@ -6,20 +6,28 @@ Continuous integration
 We use continuous integration services on GitHub to automatically check
 if new pull requests do not break anything and meet code quality
 standards such as a common `coding style <#Coding-style>`__.
+Before setting up Continuous Integration, be sure that you have set
+up your developer environment, and installed a
+`developement version <https://www.sktime.org/en/stable/installation.html>`__
+ of sktime.
 
 .. contents::
    :local:
 
 Code quality checks
 -------------------
 
-.. _precommit: https://pre-commit.com
+.. _pre-commit: https://pre-commit.com
 
-We use `pre-commit <precommit>`_ for code quality checks.
-These checks run automatically before you make a new commit.
+We use `pre-commit`_ for code quality checks (a process we also refer to as "linting" checks).  We recommend that you set this up locally as it will
```

Comment:
may be worth clarifying the difference between the linting check and the linting pre-commit?
I know it was like that in the status quo, but now looking at it I get a feeling it might not  be too clear.

* check = "is the code linted"? CI fails if it is not
* pre-commit = auto-lints the code

as a complication, the CI check uses pre-commit, and checks whether pre-commit *does not change anything*

not sure how to explain this well.

Might as well be that not changing it is easiest to understand, because "too much detail" also makes the text harder to read.

Reply:
I think a possible terminology is "check" vs "fix", i.e., "pre-commit check" vs "pre-commit fix"

Reply:
on 2nd thought, that might overcomplicate things